packages/time/package.json

```diff
@@ -41,7 +41,8 @@
   },
   "files": [
     "*.js",
-    "NEWS.md"
+    "NEWS.md",
+    "src"
```

Comment:
could we do a build instead so we're not back to exporting all the modules?

Reply:
I don't see how this is any worse than the status quo.  This package.json is not viable: it produces a tarball that doesn't have any implementation.  That broke the integration test.

What kind of build do you have in mind?  If you just care about exports, let's add `packageJson.exports`, but I don't want to tackle that in this PR.

Reply:
> I don't see how this is any worse than the status quo.

It's net better, for sure. But it fails at something that was _intended_ and I wonder if we can make it work as intended.

I had in mind a `dist/` style build but I think your `packageJson.exports` suggestion is better. I agree it can wait for another PR.